maud/src/lib.rs

```diff
@@ -343,6 +343,51 @@ mod actix_support {
     }
 }
 
+#[cfg(feature = "ntex")]
+mod ntex_support {
+    use core::{
+        error::Error,
+        mem,
+        task::{Context, Poll},
+    };
+    use ntex::{
+        http::{
+            body::{BodySize, MessageBody},
+            header, StatusCode,
+        },
+        util::Bytes,
+        web::{ErrorRenderer, HttpRequest, HttpResponse, Responder},
+    };
+
+    use crate::PreEscaped;
+    use alloc::{rc::Rc, string::String};
+
+    impl MessageBody for PreEscaped<String> {
+        fn size(&self) -> BodySize {
+            self.0.size()
+        }
+
+        fn poll_next_chunk(
+            &mut self,
+            _: &mut Context<'_>,
+        ) -> Poll<Option<Result<Bytes, Rc<dyn Error>>>> {
+            if self.0.is_empty() {
+                Poll::Ready(None)
+            } else {
+                Poll::Ready(Some(Ok(Bytes::from(mem::take(&mut self.0).into_bytes()))))
+            }
```

Comment:
Can we forward to the implementation for `String`?

```suggestion
            self.0.poll_next_chunk(context)
```